charts/newrelic-infra-operator/templates/configmap.yaml

```diff
@@ -1,8 +1,24 @@
 {{- define "fargate-config" -}}
 infraAgentInjection:
   resourcePrefix: {{ template "newrelic-infra-operator.fullname" . }}
+{{- if include "newrelic.fargate" . }}
+  agentConfig:
+{{- if not .Values.config.infraAgentInjection.agentConfig.customAttributes }}
+    customAttributes:
```

Comment:
I wonder if we should have `default-config`, including `fargate-config` template, so setting `resourcePrefix` will be done indepedently from fargate settings. That would require 3 levels of merging though.

Reply:
I would avoid adding an other level if not strictly needed 😅